src/components/__tests__/LibrariesList-test.tsx

```diff
@@ -10,27 +10,46 @@ describe("LibrariesList", () => {
   let libraries = {
     libraries:
       [{
-        "uuid": "UUID1",
-        "name": "Test Library 1",
-        "short_name": "lib1",
-        "id": 1,
-        "registry_stage": "testing",
-        "library_stage": "production"
+        uuid: "UUID1",
```

Comment:
None of this test data is used in this file, no?

Reply:
No.  But TypeScript becomes distressed if I don't have it (because the interface is expecting all of this).

Reply:
You can remove the ones that are optional.

Reply:
These are already the ones that aren't optional...